client/src/Editor/ChildList.jsx

```diff
@@ -0,0 +1,40 @@
+import React, { useEffect, useState } from "react";
+import PropTypes from "prop-types";
+import { useFetchContext } from "../context";
+import { apiUrl } from "../routes";
+import { Link } from "react-router-dom";
+
+const ChildList = ({ pid = null }) => {
+    const {
+        action: { fetchJSON },
+    } = useFetchContext();
+    const [children, setChildren] = useState([]);
+
+    useEffect(() => {
+        async function loadData() {
+            let data = [];
+            const url = apiUrl + "/edit/object/children" + (pid === null ? "" : "/" + pid);
```

Comment:
u can fetch something without a pid?

Reply:
The service provides the children of the provided pid. If you do not provide a pid, it returns the top-level objects that have no parents. I considered giving this an entirely different URL route (and am still open to the possibility), but I couldn't think of a great name for that, and using a more unified routing strategy kept things simple.